Yosemite/YosemiteTests/Stores/OrderStoreTests.swift

```diff
@@ -360,7 +360,7 @@ final class OrderStoreTests: XCTestCase {
         XCTAssertEqual(storageOrder2?.toReadOnly(), sampleOrderMutated2())
         XCTAssertEqual(viewStorage.countObjects(ofType: Storage.Order.self), 1)
         XCTAssertEqual(viewStorage.countObjects(ofType: Storage.OrderItem.self), 1)
-        XCTAssertEqual(viewStorage.countObjects(ofType: Storage.OrderItemTax.self), 5)
+        XCTAssertEqual(viewStorage.countObjects(ofType: Storage.OrderItemTax.self), 4)
```

Comment:
nice that the issue is caught in a test case!

unrelated to the PR: would be nice for the sample orders in this test case using `copy` so it's more clear what fields change :) (it's a bit hard to follow what's being upserted and its expected changes right now)